internal/commands/accesslist/inputs.go

```diff
@@ -0,0 +1,16 @@
+package accesslist
+
+// Flag names and usages across the accessList commands
+const (
+	flagIP            = "ip"
```

Comment:
looking at this, I'd like to merge my stuff and help you through the conflicts here as opposed to the other way around, it will help shape/change a few of these strings.  ultimately, this file should go away and we'll only add it back if/when a flag _does_ need to be re-used or kept consistent across these `accesslist` commands

Reply:
Sounds good!